examples/tauri.rs

```diff
@@ -0,0 +1,26 @@
+#![cfg_attr(
+  all(not(debug_assertions), target_os = "windows"),
+  windows_subsystem = "windows"
+)]
+
+use tauri::Manager;
+use window_vibrancy::{apply_blur, apply_vibrancy, NSVisualEffectMaterial};
+
+fn main() {
+  tauri::Builder::default()
+    .setup(|app| {
+      let win = app.get_window("main").unwrap();
+
+      #[cfg(target_os = "macos")]
+      apply_vibrancy(&win, NSVisualEffectMaterial::AppearanceBased)
+        .expect("Unsupported platform! 'apply_vibrancy' is only supported on macOS");
+
+      #[cfg(target_os = "windows")]
+      apply_blur(&window, Some((18, 18, 18, 125)))
```

Comment:
`window` isn't defined. I'm pretty sure you meant to use `win` right?
```suggestion
      apply_blur(&win, Some((18, 18, 18, 125)))
```

Reply:
Yes, I only have macos, sorry , I forget this PR, wait me , let me fix it and add more code in  example